pkg/xds/envoy/routes/reset_tags_configurer_test.go

```diff
@@ -0,0 +1,31 @@
+package routes_test
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	util_proto "github.com/Kong/kuma/pkg/util/proto"
+	"github.com/Kong/kuma/pkg/xds/envoy/routes"
+)
+
+var _ = Describe("ResetTagsConfigurer", func() {
+
+	It("should generate proper Envoy config",
+		func() {
```

Comment:
Checkout the formatting of different tests. It should be
`It("should generate proper Envoy config", func() {`